tools/src/main/resources/ui/js/uiutils.js

```diff
@@ -109,8 +109,8 @@ function recommendationTableCellStyle(recommendation) {
 /* define recommendation grouping */
 const recommendationRanges = {
   "A": {low: 2.5, high: 10.0},
-  "B": {low: 1.25, high: 2.5},
-  "C": {low: -1000.0, high: 1.25},
+  "B": {low: 1.3, high: 2.5},
```

Comment:
why do we have different ranges than the tool has? This would lead to confusing the user by having UI show something different then in the files.  this should be removed and just use the recommendation from the tool

Reply:
fixed the range in the tool.
Matt preferred to have single digit. So, I could not keep lower bound of 1.25

Reply:
we can remove this then right?  similar its usage this.range=recommendationRanges[id]